### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by tnhaider** [sskender/pornhub-api#27]
Hi,

Is it possible to extend the code to also retrieve comments under the videos?

Thanks

**Comment by sskender** [comment]
Yes, it should be possible with no problems. I am not working on this repo, but feel free to contribute and I will merge PR.
